src/documentation/components/Navigation.vue

```diff
@@ -19,6 +19,8 @@
           <li><NuxtLink to="/components/dropdown">Dropdown</NuxtLink></li>
           <li><NuxtLink to="/components/file-input">File input</NuxtLink></li>
           <li><NuxtLink to="/components/flag-icon">Flag icon</NuxtLink></li>
+          <li><NuxtLink to="/components/header">Header</NuxtLink></li>
```

Comment:
make sure the header one is only uncommented if header component gets merged before this one